### PR TITLE
COR:Remove duplicated code

### LINKED ISSUE #35: Remove duplicated code
두 플레이어의 이름을 입력받을 때 같을 경우 수정하기 전의 이름을 파일에 저장하는 필요없이 중복되는 statement 삭제하기

leaderboard.txt 파일을 만드는 코드가 있는데, 굳이 만들어놓지 않아도 작업을 할 때 생성되므로 중복되는 statement 삭제하기

getchar()가 너무 남발되고 있어 getchar()가 필요하지 않은 부분 삭제하기

### PR DESCRIPTION
**Pull request by seohyeonKK**
Remove the duplicated code that creates the file.
Remove the code that writes the leaderboard when both players have the
same name.
Remove useless getchar() from the code when draw or someone wins.

gechar()는 main() 내의 함수를 참고해주세요^^

### CODE REVIEW

File: Code.c [hunk 1]
```diff
@@ -54,18 +52,18 @@ int main(void)
...
     	scanf("%s", player2.name);
-    	fprintf(leaderboard, "\t%s", player2.name);
-    	fclose(leaderboard);
 
     	if(strcmp(player1.name, player2.name))
     	{
-	        chooseSymbol(&player1,&player2);
+	        leaderboard = fopen("leaderboard.txt", "a+");
+		fprintf(leaderboard, "\n%s", player1.name);
+		fprintf(leaderboard, "\t%S", player2.name);
```
Comment (mjseok) [inline Code.c]: 소문자 s로 바꿔주세요!!

**Comment by hanzzb** [comment]
소문자 s만 바꾸고 머지합시당~

**Comment by cherry2030** [comment]
merge합시다!

**Comment by mjseok** [comment]
이제 merge 합시다~!

**Pull request merged**